addons/mrp_subcontracting/models/mrp_production.py

```diff
@@ -199,3 +200,9 @@ def _get_subcontract_move(self):
 
     def _get_writeable_fields_portal_user(self):
         return ['move_line_raw_ids', 'lot_producing_id', 'subcontracting_has_been_recorded', 'qty_producing', 'product_qty']
+
+    def _get_default_finished_date(self):
```

Comment:
I would avoid all this override and directly write the date_planned_finish in _prepare_subcontract_mo_vals